### PR TITLE
Billy tags to staging

### PR DESCRIPTION
**Pull request by dmuelle**

### CODE REVIEW

File: blog_tags.json [hunk 1]
```diff
@@ -264,7 +264,7 @@
         },
         {
             "name": "community",
-            "posts": ["emilyj-eclipse",
+            "posts": ["BillyDawson_MeetTheTeam.adoc","emilyj-eclipse",
```
Comment (mbroz2) [inline blog_tags.json]:
```suggestion
            "posts": ["BillyDawson_MeetTheTeam","emilyj-eclipse",
```

File: blog_tags.json [hunk 2]
```diff
@@ -289,7 +289,7 @@
         },
         {
             "name": "developer experience",
-            "posts": ["user-feature-install",
+            "posts": ["BillyDawson_MeetTheTeam.adoc","user-feature-install",
```
Comment (mbroz2) [inline blog_tags.json]:
```suggestion
            "posts": ["BillyDawson_MeetTheTeam","user-feature-install",
```

**Review by mbroz2 — CHANGES_REQUESTED** [review]
no file extensions in blog_tags.json

**Comment by dmuelle** [comment]
fixed- thanks

**Review by mbroz2 — APPROVED** [review]

**Pull request merged**